src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -256,10 +256,8 @@ def _get_node_maps(self, cores, gpus, threads_per_proc):
         n_procs =  len(cores) / threads_per_proc
 
 
-        # core_map is a list of list of core ids.
-        # core_map = [cores_for_proc_1,cores_for_proc_2,....., cores_for_proc_N]
-        # cores_for_proc_x = [core_for_thread_1, core_for_thread_2,.., 
-        #                       core_for_thread_M]
+        # core_map is a list of list of core indexes.
+        # See Line 124-126 of ../base.py
```

Comment:
Line numbers are a fickle thing.  Maybe 'see top level comment in `./base.py`', or something like that?

Reply:
Makes sense. Done.